dangerfile.js

```diff
@@ -25,17 +25,15 @@ import fs from 'fs';
 import { includes } from 'lodash';
 import { danger, fail, warn, message } from 'danger';
 
-import * as meta from './meta.json';
-
 const AVOID_EXACT_WORDS = [
   { word: 'react native', reason: 'Please use React Native with capitals' },
 ];
 
-const BACKPACK_SQUAD_MEMBERS = meta.maintainers.map(
-  maintainer => maintainer.github,
-);
 const author = danger.github.pr.user.login;
-const isPrExternal = !BACKPACK_SQUAD_MEMBERS.includes(author);
+const isPrExternal = !danger.github.api.orgs.checkMembershipForUser({
+  org: 'backpack',
```

Comment:
We should probably be using the org `Skyscanner` as that is where the repo and its users sit rather than a specific Backpack org. We have one but that holds different things and not connected to running danger here


```suggestion
  org: 'skyscanner',
```

Reply:
Ah if we want to use Skyscanner it might be even easier :) Right now the list of maintainers matched the backpack org so I used that. We just use it for the thank you message, which might be nice for non-backpack Skyscanner employees as well?

Reply:
Yeah it would be great if we can allow it to run for any Skyscanner employees contributing changes, so they can get the danger feedback as well 😄

Reply:
With these changes too it might also mean we can completely remove the `meta.json` file as it may not be used outside of this script, and the npm sections were left over from when we split the docs from the Backpack web repo

Reply:
Yep danger runs for everybody making contributions - the `isPrExternal` check just adds a thank you message for anybody external, so if we use the backpack org we won't get the thank you, but everybody else will

Reply:
Ah yes I see now its literally just used to add that message! Am wondering if that is needed at all and we just post the message regardless of who it is? 😆

Reply:
Haha maybe! On the android project we also have a warning if it's done as a fork that CI will fail - not sure if other projects also have more checks. It did feel weird, though getting a thank you for essentially doing my job

Reply:
In our other projects we put checks in place that don't run CI steps that are dependent on things only available on the main repo (e.g. tokens for CI) so we don't give that warning as CI won't run in parts

Reply:
Just for some context, I think we did used to use it for more than just the thank you message but that need went away, so now it's only used for this trivial lil' thing